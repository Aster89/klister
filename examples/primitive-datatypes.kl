#lang kernel

(example (flip))
(example (add))
(example (remove))

(example (unit))
(example (the (Unit) (unit)))
