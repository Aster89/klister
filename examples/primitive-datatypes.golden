(flip) : ScopeAction
(add) : ScopeAction
(remove) : ScopeAction
(unit) : Unit
(unit) : Unit
